### PR TITLE
Added Next Prime Number python script

### PR DESCRIPTION
**Pull request by Tarun-Kamboj**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES.

### Description

Added a python script which calculates and print the next prime number from the number given by the user.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#3800

### CODE REVIEW

**Review by rohanbabbar04 — COMMENTED** [review]
1)This code should work for all numbers not only to a certain limit
2)Define a function and put the logic of the code there
3)Add Time and Space complexity
4)You don't need to explain each and every line with a comment(only important ones)

**Comment by Tarun-Kamboj** [comment]
> 1)This code should work for all numbers not only to a certain limit
> 2)Define a function and put the logic of the code there
> 3)Add Time and Space complexity
> 4)You don't need to explain each and every line with a comment(only important ones)

Ok

**Comment by Tarun-Kamboj** [comment]
Done @rohanbabbar04.

**Review by Tarun-Kamboj — COMMENTED** [review]
@rohanbabbar04 can you please review this now?

**Comment by Tarun-Kamboj** [comment]
@rohanbabbar04 @siddharth25pandey @anushkrishnav @ankitaggarwal23 any mentor, please review this PR.

File: Python/math/Next_Prime_Number.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+'''
+Introduction 
+
+The code below calculates and returns the next prime number from the number given by user.
+
+Time Complexity : O(n²)
+Space Complexity : O(1)
```
Comment (ankitaggarwal23) [inline Python/math/Next_Prime_Number.py]: Shift TC & SC at the EOF

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Add few more description at the top.

**Comment by Tarun-Kamboj** [comment]
Requested changes done @ankitaggarwal23. Please check.

File: Python/math/Next_Prime_Number.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+n = int(input("Enter a number : "))
+print('Next prime number is ', next_prime(n))
+
+'''
+OUTPUT
+
+Enter a number : 23
+Next prime number is  29
+
+
```
Comment (ankitaggarwal23) [inline Python/math/Next_Prime_Number.py:35]: Remove the extra one line

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Python/math/Next_Prime_Number.py (hunk 2 above)
Comment (Tarun-Kamboj) [reply Python/math/Next_Prime_Number.py:35]: It's intentional, so that TC & SC doesn't look part of the output.

Comment (Tarun-Kamboj) [reply Python/math/Next_Prime_Number.py:35]: Anything else?

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**